lib/oci_utils/oci_api.py

```diff
@@ -1028,31 +1021,17 @@ def get_volume(self, volume_id, refresh=False):
                          volume_data=vol_data,
                          attachment_data=v_att_data)
 
-    def get_compartment(self, compartment_id, refresh=False):
-        """
-        Get compartment by ID
-
-        Parameters
-        ----------
-        compartment_id : str
-            The ID of the wanted compartment.
-        refresh : bool
-            Flag, refresh the information if set.
+    def get_compartment(self, **kargs):
+        if 'ocid' not in kargs:
+            # for now make it mandatory
+            raise StandardError('ocid must be provided')
```

Comment:
This is a pretty significant change, have this been tested/validated with all the tools (including kvm)

Reply:
Sorry for confusion. 
Forgot the PR was open for review. I've push this to be able to pull it form another wks 
Testing is still ongoing.